lib/src/components/NxCollapsibleItems/NxCollapsibleItems.scss

```diff
@@ -196,41 +206,45 @@ button, a {
     width: 100%;
 
     &:hover {
-      background-color: var(--nx-swatch-indigo-95);
+      background-color: var(--nx-color-interactive-background-hover);
+      color: var(--nx-color-text-dark);
```

Comment:
You should be able to remove the `color` property here since the items are always `--nx-color-text-dark`